<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>87</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="btnAbfrageStarten">
       <property name="text">
        <string>Abfrage starten</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnZuruecksetzen">
       <property name="text">
        <string>Zurücksetzen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnWoerterbuch">
       <property name="text">
        <string>Wörterbuch</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStatistik">
       <property name="text">
        <string>Statistik</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>21</y>
      <width>127</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnNeueSprache">
       <property name="text">
        <string>Neue Sprache anlegen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNeuesBuch">
       <property name="text">
        <string>Neues Buch anlegen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNeueLektion">
       <property name="text">
        <string>Neue Lektion anlegen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNeueVok">
       <property name="text">
        <string>Neue Vokabeln eingeben</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>32</y>
      <width>111</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="btnSpracheBeareiten">
       <property name="text">
        <string>Sprachen bearbeiten</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLektionbearbeiten">
       <property name="text">
        <string>Lektionen bearbeiten</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBuecherBearbeiten">
       <property name="text">
        <string>Bücher bearbeiten</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDateu">
    <property name="title">
     <string>Datei</string>
    </property>
   </widget>
   <addaction name="menuDateu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStatistik">
   <property name="text">
    <string>Statistik</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
